test/main/deepEqual.js

```diff
@@ -1528,3 +1528,226 @@ QUnit.test( "Test that must be done at the end because they extend some primitiv
 		);
 	}
 );
+
+QUnit.module( "equiv Maps and Sets");
+// Note: we cannot do simple check for constructor existence,
+// as IE 11 implementation of Sets/Maps is incomplete.
+//  - in IE 11, Sets/Maps do not support constructors, e.g. new Set( [1] ) fails.
+//  - in IE 11, QUnit.objectType( new Set() ) === 'object'
+//
+// This is a more robust way of checking for ES6 Set/Map support.
+var hasES6Set = (function() {
+	try {
+		var s = new Set([ 1, 2, 3 ]);
+		if (s.size !== 3 || !s.has(2)) {
+			return false;
+		}
+	}
+	catch (e) {
+		return false;
+	}
+
+	return true;
+})();
```

Comment:
These flags should be conditioned to the same kind of test from QUnit.objectType, so if Set is existent, callable with a new target and `(new Set()).toString()` returns `"[object Set]"`, we can assume ES6's Set is implemented. Same thing on Map.

This way, we can also reduce the comments to say only `in IE 11, QUnit.objectType( new Set() ) === 'object'`.